server/migrations/1625814801416-CreateOrganizations.ts

```diff
@@ -3,6 +3,8 @@ import { MigrationInterface, QueryRunner, Table, TableIndex } from "typeorm";
 export class CreateOrganizations1625814801416 implements MigrationInterface {
 
   public async up(queryRunner: QueryRunner): Promise<void> {
+    
+
```

Comment:
Can you please remove the additional line break here?

Reply:
@Navaneeth-pk  Done